platform/ios/src/MGLMapView.h

```diff
@@ -272,23 +273,24 @@ MGL_EXPORT IB_DESIGNABLE
 @property (nonatomic, readonly) UIButton *attributionButton;
 
 /**
- Support for style classes has been removed. This property always returns an empty array.
+ Show the attribution and telemetry action sheet.
+
+ This method is called when the user taps on `attributionButton`. This method is
+ made available so that it can be assigned to custom attribution button
+ implementations.
  */
+- (IBAction)showAttribution;
```

Comment:
This is a valid IBAction signature, but it's customary to add a sender parameter even if we won't use it for anything. (It helps distinguish callers, sort of like the first parameter in a delegate method.)

Reply:
^ Agree. The sender parameter will make this clear.

Reply:
SGTM. → https://github.com/mapbox/mapbox-gl-native/pull/10085/commits/9d743134378b569188a2d2a69a0d193f3e66c342